src/utils/wallet.ts

```diff
@@ -242,6 +252,159 @@ export const groomingWalletData = async (user, globalProps, userCurrency) => {
   return walletData;
 };
 
+
+export const fetchCoinsData = async (
+  coins:CoinBase[], 
+  username:string, 
+  vsCurrency:string, 
+  globalProps:GlobalProps
+  ) 
+  : Promise<{[key:string]:CoinData}> => {
+
+    const {base, quote, hivePerMVests} = globalProps
+
+    const coinData = {} as {[key:string]:CoinData};
+    const walletData = {} as any;
+
+
+    if (!username) {
+      return walletData;
+    }
+
+    //TODO: Use already available accoutn for frist wallet start
+    const state = await getAccount(username);
+    const ecencyUser = await getEcencyUser(username)
+    //TODO: cache data in redux or fetch once on wallet startup
+    const ppHbd = await getCurrencyTokenRate(vsCurrency, 'hbd');
+    const ppHive = await getCurrencyTokenRate(vsCurrency, 'hive');
+    const ppEstm = await getCurrencyTokenRate(vsCurrency, 'estm');
```

Comment:
@feruzm this method returns estm token value to be 0.002, however, I remember equating 1 point to 1/150 usd while the returned value equates to 1/500... which should I use for Ecency points calculation...?

![Screenshot 2022-02-20 at 9 49 21 AM](https://user-images.githubusercontent.com/6298342/154828938-02b0539d-1a9b-4d4c-8ea3-4ec7388be58a.png)

Reply:
@noumantahir there is purchase value and there is content boosting value, they are different.